frontends/web/src/locales/en/app.json

```diff
@@ -579,6 +579,7 @@
         "label": "microSD card",
         "true": "Inserted"
       },
+      "securechip": "Secure chip",
```

Comment:
Alternatively, "Secure element"? The "chip" is more nerdy and I like it  :) but I've seen this alt name in many places. Not sure though.

Reply:
We've used secure chip everywhere since the beginning, e.g. here: https://shiftcrypto.ch/bitbox02/